welcome/index.adoc

```diff
@@ -317,3 +317,18 @@ endif::[]
 endif::openshift-enterprise,openshift-webscale,openshift-origin[]
 
 endif::openshift-rosa[]
+
```

Comment:
Hi @lahinson .

Will updates also be made to the OCP 4.14. Release notes? I see numerous comments in the `ocp-4-14-release-notes.adoc` file that suggests updates on the following lines in the file:

* 375
* 412
* 1281
* 2683
* 2688
* 
Or search for "Hosted Control Planes" in the event the line numbers have changed.

Reply:
Hi @dfitzmau - Yes, my updates to the 4.14 release notes are ongoing in this PR: https://github.com/openshift/openshift-docs/pull/66219

I believe @SNiemann15 will also have updates to the 4.14 release notes related to the upcoming IBM Power and IBM Z docs for hosted control planes.

Reply:
Also, it looks like the RHACM GA is slipping a bit, so the docs won't be live tomorrow (Nov. 15). I am waiting to find out what the new GA date will be.